packages/editorconfig/src/browser/editorconfig-document-manager.ts

```diff
@@ -98,6 +98,7 @@ export class EditorconfigDocumentManager {
             const uri = editor.uri.toString();
             this.editorconfigServer.getConfig(uri).then(properties => {
                 this.properties[uri] = properties;
+                this.applyProperties(properties, editor);
```

Comment:
@svenefftinge Did you remove it on purpose?

Reply:
I think this change is ok, otherwise settings like [`end_of_line`](https://github.com/editorconfig/editorconfig/wiki/Plugin-Guidelines#end_of_line) won't work as expected.

Reply:
pl